packages/protocol/contracts/governance/DowntimeSlasher.sol

```diff
@@ -83,7 +83,7 @@ contract DowntimeSlasher is SlasherUtil {
       uint256 signerIndex = epochNumberOfBlock(n, sz) == startEpoch
         ? startSignerIndex
         : endSignerIndex;
-      if (uint256(getParentSealBitmap(n)) & (1 << signerIndex) != 0) return false;
+      if (uint256(getParentSealBitmap(n + 1)) & (1 << signerIndex) != 0) return false;
```

Comment:
So, not sure of the semantics of `getParentSealBitmap`. 
From your change it seems that `getParentSeal(x)` will get the parentSeal that's on block X header. which is a seal of all signers for block X-1.

In that case, this is correct.. and great catch by the way!
Can you confirm? The other case is that getParentSealBitmap(x) get's the parent seal that have signer for block X and that lives on block X+1 header..

In any case, a comment would be nice :D

Reply:
I think I'll check from geth, but I tried to check from baklava, the current version didn't seem to work.

Reply:
It gets the header from the block given as argument https://github.com/celo-org/celo-blockchain/blob/master/core/vm/contracts.go#L797

So it should be correct now.